mobile/mobile.go

```diff
@@ -493,6 +493,25 @@ func (m *Mobile) GetPhotoMetadata(id string) (string, error) {
 	return toJSON(meta)
 }
 
+// ResolveProfileInfo take a peer id and profile key name, like "username"
+func (m *Mobile) ResolveProfileInfo(peerId string, key string) (string, error) {
```

Comment:
^ here's the method for getting profile keys like username, and avatar_id under a peer's id